code/modules/busy_space/organizations_vr.dm

```diff
@@ -224,3 +224,32 @@
 						"a settlement needing our help",
 						"Forward Base Sigma-Alpha in ArCon space")
 
+/datum/lore/organization/gov/Scion
+    name = "Scion Imperium"
+    short_name = "Scion"
+    desc = "A Imperium of Desert Elves, slowly expanding and exploring the galaxy on there own terms instead of those of others./
```

Comment:
A few typos, mostly there/there mixups in the description. Should read "on their own terms", "Their ships are..." and "Despite their lack of ships, their ground forces..."
History section misspells "calendar".